internal/driver/glfw/window_desktop.go

```diff
@@ -261,26 +261,89 @@ func (w *window) fitContent() {
 	}
 }
 
+// monitorInfo contains the information needed for monitor detection
+type monitorInfo struct {
+	X, Y, Width, Height int
+}
+
+// findMonitorWithLargestIntersection finds the monitor with the largest intersection area
+// with the given window bounds. Returns the index of the best monitor, or -1 if no intersection.
+// This is extracted for testing purposes.
+func findMonitorWithLargestIntersection(windowX, windowY, windowWidth, windowHeight int, monitors []monitorInfo) int {
+	windowLeft := windowX
+	windowRight := windowX + windowWidth
+	windowTop := windowY
+	windowBottom := windowY + windowHeight
+
+	var bestMonitorIndex int = -1
+	var bestArea int
+
+	for i, mon := range monitors {
+		// Calculate monitor bounds
```

Comment:
The comments inside this method are not needed - the code is clean and clear.

Reply:
Code removed in 7f112e390cdaa728c9bc3134e7b7187ed4a67261.